packages/flutter_tools/lib/src/isolated/web_server_utlities.dart

```diff
@@ -0,0 +1,55 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:logging/logging.dart' as logging;
+import 'package:package_config/package_config.dart';
+
+import '../base/file_system.dart';
+import '../base/logger.dart';
+import '../dart/package_map.dart';
+import '../web_template.dart';
+
+void log(Logger logger, logging.LogRecord event) {
```

Comment:
These now public functions need a little bit of documentation. All the methdos are pretty straight-forward and self-explanatory, and their docs can be simple. For example, this one could be:

```
/// Logs [event] to [logger].
///
/// The `event.level` property determines whether the event will
/// be logged as an Error (SEVERE), a warning (WARNING) or a trace
/// (everything else).
```